profiles/base.yaml

```diff
@@ -225,10 +225,16 @@ loot_delay: 5
 dump_junk: false
 # items to wait for before junking
 dump_item_count: 9
-# used to set a pause timer for barbarian meditations, this number should go down with a chakrel, circles, and stats...
-# as an example, chuno currently runs it at 6 being circle 195
-# defaulting to 25, as 20 for lower skilled barbs was explained to be common
-meditation_pause_timer: 25
+# Used to set a pause timer for barbarian meditations.
+# This number should go down with a chakrel, circles, and stats...
+# When buff script is activating meditations,
+# this is how many seconds to wait for it to activate
+# before moving on to the next action.
+meditation_pause_timer: 6
```

Comment:
Reduced `meditation_pause_timer` in base.yaml to 6 seconds from 25 seconds.

As a 30th circle barbarian, I haven't noticed taking more than 6 seconds for Tenacity meditation to take effect.

The SampleBarbarian-setup.yaml uses 6 seconds, too.

And combat-trainer was only waiting a hard coded 6 seconds if it activated a meditation, too.

Other than this setting being introduced to combat-trainer (rather than the pause amount being hard coded), this setting was only used elsewhere in common-arcana's `start_barb_abilities` method, and that method isn't used anywhere in the Lich codebase. Therefore, I don't foresee any risk lowering the default value.